src/redux/utils.ts

```diff
@@ -103,6 +103,10 @@ export function invalidateCache(action: string, dispatch: Function) {
 }
 
 export function errorMsgResponse(response: any) {
+  let errorMsgResponse = { __detail__: "Action could not be completed. Please retry." };
+
+  if (response.data && has(response.data, "detail")) {
+    errorMsgResponse["__detail__"] = response.data.detail
```

Comment:
The error in the build in Travis is caused by this line... essentially the `{` after the `if` is never closed. That said we don't need this `if` here at all. What you have on line 109 should be in place of what's on 117 and line 120-124 should look something like:

```ts
errorMsgResponse[key] =
  Array.isArray(response.data[key])
    ? response.data[key].join(",")
    : response.data[key];
```